src/api/endpoints/logged_activities/crud.py

```diff
@@ -72,6 +74,20 @@ def post(self):
                     ]
 
             logged_activity.save()
+            society_id = g.current_user.society_id
+
+            #send notification to the society secretary about logged points
+
+            roles = User.query.filter(User.roles.any(Role.name=="society secretary")).all()
```

Comment:
Could we make this a class variable, instead of calling it in every POST request?

Reply:
Roles change dynamically, making this a class variable will miss out on recently added `society secretaries`.

Reply:
We could do the queryset thing then filter on every request to get the list.

Reply:
This joins `user` and `role` table with the `user-role` relationship. Keep in mind that the user-role relationship table has no `id` column.